web/html/src/manager/proxy/container-config.tsx

```diff
@@ -0,0 +1,497 @@
+import * as React from "react";
+
+import { AsyncButton } from "components/buttons";
+import { MessageType } from "components/messages";
+import { TopPanel } from "components/panels/TopPanel";
+
+import { ContainerConfigMessages } from "./container-config-messages";
+
+enum SSLMode {
+  UseSSL = "use-ssl",
+  CreateSSL = "create-ssl",
+}
+
+type Props = {};
+
+type State = {
+  caCertificate?: File;
+  caKey?: File;
+  caPassword: string;
+  country: string;
+  state: string;
+  city: string;
+  org: string;
+  orgUnit: string;
+  sslEmail: string;
+  rootCA?: File;
+  intermediateCA?: File;
+  proxyCertificate?: File;
+  proxyKey?: File;
+  email: string;
+  sslMode: SSLMode;
+  loading: boolean;
+  maxSquidCacheSize: string;
+  messages: MessageType[];
+  proxyFQDN: string;
+  success?: any;
+};
+
+export class ProxyConfig extends React.Component<Props, State> {
+  initState: State;
+
+  constructor(props: Props) {
+    super(props);
+
+    this.initState = {
+      caCertificate: undefined,
+      caKey: undefined,
+      caPassword: "",
+      country: "",
+      state: "",
+      city: "",
+      org: "",
+      orgUnit: "",
+      sslEmail: "",
+      rootCA: undefined,
+      intermediateCA: undefined,
+      proxyCertificate: undefined,
+      proxyKey: undefined,
+      email: "",
+      sslMode: SSLMode.CreateSSL,
+      loading: false,
+      maxSquidCacheSize: "",
+      messages: [],
+      proxyFQDN: "",
+    };
+
+    this.state = this.initState;
+  }
+
+  onSubmit = () => {
+    this.setState({ messages: [], loading: true });
+    var formData: any = {};
+    formData["proxyFQDN"] = this.state.proxyFQDN;
+    formData["maxSquidCacheSize"] = this.state.maxSquidCacheSize;
+    formData["email"] = this.state.email;
+    formData["sslMode"] = this.state.sslMode;
+
+    if (this.state.sslMode === SSLMode.CreateSSL) {
+      formData["caCertificate"] = this.state.caCertificate;
+      formData["caKey"] = this.state.caKey;
+      formData["caPassword"] = this.state.caPassword;
+      formData["country"] = this.state.country;
+      formData["state"] = this.state.state;
+      formData["city"] = this.state.city;
+      formData["org"] = this.state.org;
+      formData["orgUnit"] = this.state.orgUnit;
+      formData["sslEmail"] = this.state.sslEmail;
+    } else {
+      formData["rootCA"] = this.state.rootCA;
+      formData["intermediateCA"] = this.state.intermediateCA;
+      formData["proxyCertificate"] = this.state.proxyCertificate;
+      formData["proxyKey"] = this.state.proxyKey;
+    }
+    const request = Network.post(
+      "/rhn/manager/proxy/container-config",
```

Comment:
This should rather be `/manager/api/proxy/container-config` since this is the endpoint for submit